boards/mips/qemu_malta/qemu_malta_option.yaml

```diff
@@ -0,0 +1,4 @@
+qemu:
+  qemu_option:
+    - "-serial null"
+    - "-serial null"
```

Comment:
Duplicated code?

Reply:
convert it from the board.cmake file for this board, see below

`
set(QEMU_FLAGS_${ARCH}
  -machine malta
  -nographic
  -serial null
  -serial null
  )
`

Reply:
@mbolivar-ampere any more concens on this?